Ginger/GingerCore/GeneralLib/General.cs

```diff
@@ -906,6 +906,43 @@ public static T TryFindParent<T>(DependencyObject child)
             else
                 return TryFindParent<T>(parentObject);
         }
+
+        /// <summary>
+        /// This function is used to replace the password value from the string with *****
+        /// </summary>
+        /// <param name="dataString">The string argument which has password value</param>
+        /// <returns></returns>
+        public static string HidePasswordFromString(string dataString)
+        {
+            string passwordValue = dataString.Replace(" ", "");//remove spaces
+            string passwordString = string.Empty;
+            //Matching string
+            if (dataString.ToLower().Contains("pwd="))
```

Comment:
did you confirmed that no exception will be trowen in case the string do not contains both "Pwd" or "Password"?

Reply:
Returning original string if it does not contain matching string("pwd", or "password")